aspnet-core/Signature/overview.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: overview | Signature | ASP.NET Core | Syncfusion
-description: overview
+description: Learn more about using ASP.NET core Signature to customize the background, stroke width and stroke color, convert captured signature to an image format.
```

Comment:
Core - starting letter should be capital. change here.

And change the description like as below,

Learn more about how to customize the ASP.NET Core Signature background, stroke width and stroke color and convert captured signature to an image format.

Reply:
made changes